src/nile/common.py

```diff
@@ -134,6 +137,13 @@ def get_hash(contract_name, overriding_path=None):
     )
 
 
+def get_account_hash(contract="Account"):
+    """Return the class_hash of an Account contract."""
+    pt = os.path.dirname(os.path.realpath(__file__)).replace("/core", "")
+    overriding_path = (f"{pt}/artifacts", f"{pt}/artifacts/abis")
+    return int(get_hash(contract, overriding_path=overriding_path), 16)
```

Comment:
```suggestion
    return int(get_class_hash(contract, overriding_path=overriding_path), 16)
```
There are a lot of hashes in our scope, I think renaming `get_hash` to `get_class_hash` makes reading the code easier.

Reply:
Also, `get_hash` should return an int following the hex/int pattern (I changed these two things in the PR for deploying through accounts).

Reply:
I would rename `get_account_hash` to `get_account_class_hash` accordingly.

Reply:
the [last understanding](https://github.com/OpenZeppelin/nile/issues/258) was that `get_hash` was considered part of the external API.

In any case I think this pattern is starting to show some flaws: it happens a lot that "external" functions are used internally, and it's also very hard to know in advance whether a function is/will be external or not. It's not something you can derive from a functions name, module, etc. It's very hard to be mindful of the pattern. I wonder if we can do something to fix it.

Reply:
In this case, it doesn't matter if it is internal or external API. The pattern is to return always int unless the function name explicitly declares the intention of returning a hex.